ga4gh/frontend.py

```diff
@@ -451,7 +457,11 @@ def handleFlaskPostRequest(flaskRequest, endpoint):
     Invokes the specified endpoint to generate a response.
     """
     if flaskRequest.method == "POST":
-        return handleHttpPost(flaskRequest, endpoint)
+        try:
```

Comment:
@jeromekelleher hi. thanks for your review.  I had this here because underlying exceptions & traces seem to be swallowed up and not logged.  This helped us isolate underlying problems.  I'll remove it.  We may want to revisit in the future if others report similar.